### DISCUSSION TITLE
Create a thumbnail

### DISCUSSION THREAD

**Discussion by kKerem** [rbuchberger/jekyll_picture_tag#296] [General]
I couldn't figure how can we create a thumbnail preset based on only max-height:300px. Can anyone give me an example?

**Comment by rbuchberger** [comment]
I'm assuming that's a media query, which will only apply to screens which are less than 300px? That won't apply to very many devices, basically smaller phones in landscape mode and smartwatches or something. 

Could you give me a little more information on what you're trying to do?
